lib/card.rb

```diff
@@ -0,0 +1,13 @@
+class Card
+  attr_reader :question,:answer, :category, :prompt
```

Comment:
I'm not sure what `:prompt` is for. Maybe leftover from something you were trying out? In any case, it doesn't appear to be used for anything so it can/should be removed.
```suggestion
  attr_reader :question, :answer, :category
```